src/merge.ts

```diff
@@ -302,8 +316,9 @@ export function mergeWithMapping(inFilesContent: string | string[], destFileCont
 
     const mergedContent = xmlDeclaration + revertApostrophes(destDoc.toString({
         preserveWhitespace: true,
-        compressed: true
-    }), !options?.replaceApostrophe);
+        compressed: false
+    }), !options?.replaceApostrophe).replace(/^\s*[\r\n]/gm, '');
```

Comment:
I'd rather exclude any formatting from this lib. It should - as much as possible - keep all formattings as before. In case you'd need some formatting, I'd recommend using another tool before or afterwards - e.g. https://github.com/daniel-sc/xml_normalize . Would that work for you?

Reply:
I had the same inention. In fact you do formatting because you use `toString` with `compressed: true`

Before my changes: 
![image](https://github.com/daniel-sc/xliff-simple-merge/assets/3378880/6b05b83c-f607-4f85-9e50-efc8d30f907f)
after my changes:
![image](https://github.com/daniel-sc/xliff-simple-merge/assets/3378880/f35b294a-1f94-4dba-8245-105e1ef2e22d)

I've used the lib on a well formatted xlf file and everything was changed.